prepare_artifacts.sh

```diff
@@ -22,9 +22,9 @@ cd $CILIUM_DIR/install/kubernetes
 helm package --destination "$CWD" cilium
 cd -
 helm repo index . --merge index.yaml
-$EDITOR README.md
+./generate_readme.sh > README.md
 git add README.md index.yaml cilium-$VERSION.tgz
-git commit -s -m "Add $VERSION@$(cd $CILIUM_DIR; git rev-parse HEAD) ⎈"
+git commit -s -m "Add cilium $VERSION@$(cd $CILIUM_DIR; git rev-parse HEAD) ⎈"
```

Comment:
Added `cilium` here to make it easy to distinguish between cilium and tetragon commits. made the corresponding change in conformance-gke.yaml.